cli/jobs/train/r/basic-train-model/basic_r_job.yml

```diff
@@ -0,0 +1,17 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandJob.schema.json
+description: This job trains an rpart model on the iris dataset 
```

Comment:
match description with other jobs - follow order of job sections defined in contributing guidelines